src/Shell/Task/ModelTask.php

```diff
@@ -694,7 +701,11 @@ public function getRules($model, array $associations)
         }
 
         foreach ($associations['belongsTo'] as $assoc) {
-            $rules[$assoc['foreignKey']] = ['name' => 'existsIn', 'extra' => $assoc['alias']];
+            if (is_array($assoc['foreignKey'])) {
+                $rules[join($assoc['foreignKey'], ';')] = ['name' => 'existsIn', 'extra' => $assoc['alias']];
```

Comment:
I hope this convention is fine with you (similar happens with selects and composite primary keys by default in CakePHP3's FormHelpers)